### ISSUE TITLE
Set API Proxy description on pom.xml

### ISSUE THREAD

**Issue by vitor-a-pinto** [apigee/apigee-deploy-maven-plugin#186]
With would be nice if we could set the proxy description on the pom.xml. As far I understood the plugin tries to use the description on <proxyName>.xml file. On the new proxy bundle format this xml might not exist, so it would be nice at least to be able to set it on the pom.

**Comment by ssvaidyanathan** [comment]
@vitor-a-pinto - I am assuming you already have the <proxy>.xml in your source code and you just want the description tag of that file to be customized?

**Comment by vitor-a-pinto** [comment]
Hi  @ssvaidyanathan. On the new Apigee Proxy Bundle Archive the <proxy>.xml isn't required. VSCode plugin for GCloud\Apigee, for example, doesn't generate the <proxy>.xml file, because is compliant with the Archive Bundle. So, no, I don't have the <proxy>.xml available.

**Comment by ssvaidyanathan** [comment]
Yes - I am aware of that. It wasnt needed in the earlier version as well. But unfortunately thats the only source where you can add/update the description of the proxy. So in your case, you will have to include that in your source code (somehow) and then have the plugin package it and deploy it. Don't think the plugin should be generating it.

**Comment by vitor-a-pinto** [comment]
Ok @ssvaidyanathan  I see your point. You may close the issue.
